ocaml/parsing/parser.mly

```diff
@@ -186,6 +186,15 @@ let mktyp_local_if p typ loc =
 let wrap_exp_local_if p exp loc =
   if p then wrap_exp_stack exp (make_loc loc) else exp
 
+let exclave_ext_loc loc = mkloc "extension.exclave" loc
+
+let exclave_extension loc =
+  Exp.mk ~loc:Location.none
+    (Pexp_extension(exclave_ext_loc loc, PStr []))
```

Comment:
I think it'll be better if you give the `Exp` the same `loc` but ghostified.  On the other hand, `mkexp_stack` doesn't, so maybe it doesn't matter much?

Reply:
I recall having issues with some tooling, but I think it was fixed; let's say that what's good for `mkexp_stack` is good for `exclave_extension`